md/create-application.md

```diff
@@ -2,44 +2,56 @@
 
 Creating the application is the final step of this Getting Started tutorial.
 
-But before we actually create the application, we need to deploy our application page in the Bonita Portal:
-1. In the Bonita Studio menu click on **File > Deploy**
+To create and deploy our first application in our local environment, the easiest way is to use the **[Application descriptor](applicationCreation.md)**. This editor helps you defining the skeleton of an application, as it contains references to resources installed in Bonita Portal and used by a given application. It will also help us deploying the application directly from Bonita Studio. 
```

Comment:
```suggestion
To create and deploy our first application in our local environment, we will use the **[Application descriptor](applicationCreation.md)**. This editor helps you defining the skeleton of an application, as it contains references to resources installed in Bonita Portal and used by a given application. It will also help us deploying the application directly from Bonita Studio. 
```